lib/tasks/draft_document_collection_writer.rake

```diff
@@ -0,0 +1,16 @@
+require_relative "../../app/validators/gov_uk_url_validator"
+
+desc "Create a draft document collection in the whitehall database, from a given specialist topic"
+task :create_draft_document_collection, %i[specialist_topic_base_path assignee_email_address] => :environment do |_task, args|
```

Comment:
Ah! I'd been reviewing this PR commit-by-commit and I'd assumed you would be running this all as a bulk/batch thing. Hence my comments about performance optimisations.

Performance seems much less likely to be a concern if this only runs one-by-one! 🙂

Reply:
@hannako yep – all good!